neon_utils/log_utils.py

```diff
@@ -126,3 +139,14 @@ def get_log_file_for_module(module_name: Union[str, list]) -> str:
         log_name = "extras.log"
 
     return os.path.join(LOG_DIR, log_name)
+
+
+def init_log_for_module(service: ServiceLog, std_out: bool = False):
+    """
+    Initialize `LOG` singleton for the specified service in this thread
+    Args:
+        service: service requesting a logger object
+        std_out: if true, print logs to std_out instead of to files
+    """
+    log_file = "stdout" if std_out else os.path.join(LOG_DIR, service.value)
+    LOG.init({"path": log_file})
```

Comment:
can we expose all the config options?

see https://github.com/OpenVoiceOS/ovos_utils/blob/dev/ovos_utils/log.py#L45